components/frontend/src/semantic_ui_react_wrappers/Icon.js

```diff
@@ -1,8 +1,21 @@
+import { string } from "prop-types"
 import { useContext } from "react"
 import { Icon as SemanticUIIcon } from "semantic-ui-react"
 
 import { DarkMode } from "../context/DarkMode"
 
 export function Icon(props) {
-    return <SemanticUIIcon inverted={useContext(DarkMode)} {...props} />
+    let { className, name, ...otherProps } = props
+    /* Using name="linegraph" results in "Invalid prop `name` of value `linegraph` supplied to `Icon`."
```

Comment:
ouch, that's a sneaky.

Reply:
Yeah, but it works *and* suppresses the log message, so best possible result :-)